libc/test/src/math/smoke/ILogbTest.h

```diff
@@ -9,7 +9,7 @@
 #ifndef LLVM_LIBC_TEST_SRC_MATH_ILOGBTEST_H
 #define LLVM_LIBC_TEST_SRC_MATH_ILOGBTEST_H
 
-#include "src/__support/CPP/limits.h" // INT_MAX
```

Comment:
Was unused. I did similar cleanups in test templates that I edited in previous PRs, but I don't think I ever left a comment explaining it.